### PR TITLE
updated about page

### PR DESCRIPTION
**Pull request by taliabahar**
I added our group name to the titles and wrote a little about us. I'll add to it more as the weeks go on and we make the chat app more personalized and get to know each other better. (I also left the original notes I'll delete them later just in case we want to refer to them or incorporate them into the about description)

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -36,10 +36,15 @@
     <div
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
-      <h1>About the CodeU Chat App</h1>
+      <h1>About the Baker's Dozen CodeU Chat App</h1>
       <p>
-        This is an example chat application designed to be a starting point
-        for your CodeU project team work. Here's some stuff to think about:
+        The Baker's Dozen Chat App began as a Google CodeU chat app template from which we built
+        off of throughtout the 2018 Summer. Born and bred on the West Coast with a passion for Computer Science and problem solving,
```
Comment (lukemujica) [inline src/main/webapp/about.jsp]: throughtout --> throughout

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -36,10 +36,15 @@
...
+        the Baker's Dozen team consists of Los Angeles natives Gaby Cortes and Talia Bahar and Bay Area native Luke Mujica.
+        This Chat App represents a combinaition of the unique skill sets we came into the prgram with in conjunction with the
```
Comment (lukemujica) [inline src/main/webapp/about.jsp]: combinaition --> combination

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -36,10 +36,15 @@
...
+        new knlowdge we gained from our Google mentors (Shoutout Edrei Chua), peers, and eachother.
```
Comment (lukemujica) [inline src/main/webapp/about.jsp]: knlowdge --> knowledge

File: src/main/webapp/about.jsp [hunk 4]
```diff
@@ -36,10 +36,15 @@
...
+
+        Here's some stuff to think about:
```
Comment (lukemujica) [inline src/main/webapp/about.jsp]: What's this for? Just curious

Comment (taliabahar) [reply src/main/webapp/about.jsp]: I just added it to set up what would later be an explanation of what we added to the app

**Comment by edreichua** [comment]
Do a git pull origin master, followed by a git push origin TaliaDevelopment, to get the most recent changes!

File: src/main/webapp/about.jsp [hunk 5]
```diff
@@ -36,10 +36,15 @@
...
-      <h1>About the CodeU Chat App</h1>
+      <h1>About the Baker's Dozen CodeU Chat App</h1>
       <p>
-        This is an example chat application designed to be a starting point
-        for your CodeU project team work. Here's some stuff to think about:
+        The Baker's Dozen Chat App began as a Google CodeU chat app template from which we built
+        off of throughout the 2018 Summer. Born and bred on the West Coast with a passion for Computer Science and problem solving,
+        the Baker's Dozen team consists of Los Angeles natives Gaby Cortes and Talia Bahar and Bay Area native Luke Mujica.
+        This Chat App represents a combination of the unique skill sets we came into the prgram with in conjunction with the
+        new knowledge we gained from our Google mentors (Shoutout Edrei Chua), peers, and eachother.
```
Comment (edreichua) [inline src/main/webapp/about.jsp]: Nit: eachother -> each other

**Comment by edreichua** [comment]
LGTM

**Pull request merged**